### DISCUSSION TITLE
역을 파괴하는 이벤트의 처리

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#17] [Ideas]
태풍 등의 이벤트에 역이 파괴될 수 있습니다.
이 때 
- 해당 역을 제거할 지, 역을 비활성화 할 지
- 해당 역에 대하여 들어오고, 나가는 열차에 대한 처리
- 해당 역을 점유중인 승객에 대한 처리

위 사안에 대한 의견 바랍니다.

**Comment by seong0929** [comment]
* 역이 파괴될 경우 해당 역을 비활성화
* 파괴된 역이 연결된 노선에 기차는 다닐 수 있지만 해당 역에는 정차하지 않는다.
* 파괴된 역은 승객이 없어지고, 하루에 불만도가 5씩 오르도록 한다.
* 추가
  * 파괴된 역의 경우 제거할 수 있는 옵션을 추가
  * 역을 제거할 경우 해당 역이 있는 노선의 다른 모든 역에 일시적으로 불만도 30을 추가한다.
  * 파괴된 역을 복구할 경우 : 일정 비용을 지불하고 재활성화 가능. 재활성화 시 역 파괴로 인해 늘어났던 불만도가 사라짐.
